tests/src/CompilerTest.php

```diff
@@ -701,27 +703,34 @@ public function testMatchPropertyPath()
     
     public function testFailParameterPath()
     {
-        $scenario = self::getRelationshipsScenario();
+        // we're expecting an exception
+
+        // the api method call
+        $scenario = json_decode(self::getRelationshipsScenario(), true);
```

Comment:
We could use this:
$scenario = self::getRelationshipsScenario();

...and then, in the "verify" method, we could decode the JSON once. That way, we would never duplicate any code.